### ISSUE TITLE
Expand  env vars in tri.native.read/tri.native.write

### ISSUE THREAD

**Issue by glacambre** [tridactyl/tridactyl#2134]
Tridactyl version: 1.17.1pre3444-a0da8d5
Firefox version: Mozilla Firefox 72.0.1
Operating system: linux

```
:js tri.native.write("$HOME/hello", "world")
```

Does not work. I think the read and write functions would be more useful if Tridactyl expanded environment variables before trying to read/write things. But maybe it's dangerous to do that since read/write's input will often be input coming directly from the page? This requires more thought.

**Comment by bovine3dom** [comment]
I don't think this is a security concern - you can already get to 99% of native users' home directories with the right number of .. since Tridactyl's messenger is usually installed in the same place.

**Comment by Rummskartoffel** [comment]
If you have access to `native.write()` you have access to `native.run()`, so it really doesn't matter in terms of security. Could be a footgun, though...

**Comment by bovine3dom** [comment]
Perhaps we should have an expandpath function that replaces env variables with their values? It wouldn't be portable though so I'm not sure I see the benefit compared to `! echo $HOME` etc
